azure-pipelines-1es.yml

```diff
@@ -0,0 +1,126 @@
+            artifactName: 'TestResults'
+            publishLocation: Container
+          - output: pipelineArtifact
+            displayName: 'Publish Packages'
+            condition: succeeded()
+            targetPath: '$(Build.SourcesDirectory)\artifacts\packages\$(BuildConfiguration)'
+            artifactName: 'PackageArtifacts'
+          - output: pipelineArtifact
+            displayName: 'Publish Asset Manifests'
+            condition: succeeded()
+            targetPath: '$(Build.SourcesDirectory)/artifacts/log/$(BuildConfiguration)/AssetManifest'
+            artifactName: AssetManifests
+          - output: pipelineArtifact
+            displayName: 'Publish MicroBuild Artifacts'
+            condition: succeededOrFailed()
+            targetPath: '$(Build.ArtifactStagingDirectory)\MicroBuild\Output'
+            artifactName: MicroBuildOutputs
+            publishLocation: Container
+        steps:
+        - task: PowerShell@2
+          displayName: Setup Private Feeds Credentials
+          inputs:
+            filePath: $(Build.SourcesDirectory)/eng/common/SetupNugetSources.ps1
+            arguments: -ConfigFile $(Build.SourcesDirectory)/NuGet.config -Password $Env:Token
+          env:
+            Token: $(dn-bot-dnceng-artifact-feeds-rw)
+        - script: eng\common\CIBuild.cmd -configuration $(BuildConfiguration) /p:OfficialBuildId=$(Build.BuildNumber) /p:VisualStudioDropName=$(VisualStudioDropName) /p:DotNetSignType=$(SignType) /p:DotNetFinalVersionKind=$(_additionalBuildArgs) /p:DotnetPublishUsingPipelines=true
+          displayName: Build
+        - template: /eng/common/templates-official/steps/generate-sbom.yml@self
+        - task: PublishTestResults@2
+          displayName: Publish XUnit Test Results
+          inputs:
+            testRunner: XUnit
+            testResultsFiles: '$(Build.SourcesDirectory)\artifacts\TestResults\$(BuildConfiguration)\*.trx'
+            mergeTestResults: true
+            testRunTitle: 'Unit Tests'
+          condition: always()
+      - job: codeql
+        displayName: CodeQL
+        variables:
+        - name: skipComponentGovernanceDetection
+          value: true
+        - name: Codeql.Enabled
+          value: true
+        - name: Codeql.Cadence
+          value: 0
+        - name: Codeql.TSAEnabled
+          value: true
+        steps:
+        - task: CodeQL3000Init@0
+          displayName: CodeQL Initialize
+        - task: PowerShell@2
+          displayName: Setup Private Feeds Credentials
+          inputs:
+            filePath: $(Build.SourcesDirectory)/eng/common/SetupNugetSources.ps1
+            arguments: -ConfigFile $(Build.SourcesDirectory)/NuGet.config -Password $Env:Token
+          env:
+            Token: $(dn-bot-dnceng-artifact-feeds-rw)
+        - script: eng\common\cibuild.cmd -configuration Release -prepareMachine /p:Test=false
+          displayName: Windows Build
+        - task: CodeQL3000Finalize@0
+          displayName: CodeQL Finalize
```

Comment:
I think CodeQL Initialize and Finalize steps should be removed, they are now injected automatically from 1ES pipeline.